packages/hash/frontend/src/lib/org.ts

```diff
@@ -43,42 +50,17 @@ export type Org = MinimalOrg & {
 
 export const constructOrg = (params: {
   subgraph: Subgraph;
-  orgEntityId: string;
+  orgEntityEditionId: EntityEditionId;
 }): Org => {
-  const { subgraph, orgEntityId } = params;
-
-  const incomingOfOrgLinks = getIncomingLinksOfEntity({
-    entityId: orgEntityId,
-    subgraph,
-    linkTypeId: types.linkType.ofOrg.linkTypeId,
-  });
-
-  const orgMemberships = incomingOfOrgLinks.map(
-    ({ inner }) =>
-      subgraph.vertices[inner.sourceEntityId] as unknown as EntityVertex,
-  );
+  const { subgraph, orgEntityEditionId } = params;
 
   return {
-    ...constructMinimalOrg({ orgEntityId, subgraph }),
-    members: orgMemberships.map(({ inner: orgMembershipEntity }) => {
-      const responsibility: string =
-        orgMembershipEntity.properties[
-          extractBaseUri(types.propertyType.responsibility.propertyTypeId)
-        ];
-
-      const incomingHasMembershipLinks = getIncomingLinksOfEntity({
-        entityId: orgMembershipEntity.entityId,
-        subgraph,
-        linkTypeId: types.linkType.hasMembership.linkTypeId,
-      });
-
-      const userEntityId = incomingHasMembershipLinks[0]!.inner.sourceEntityId;
-
-      return {
-        ...constructMinimalUser({ subgraph, userEntityId }),
-        responsibility,
-      };
+    ...constructMinimalOrg({
+      subgraph,
+      orgEntityEditionId,
     }),
+    /** @todo implement incoming link subgraph helper */
```

Comment:
I'm not sure it's as simple as implementing the helper. I don't think we can query for incoming links right now

Reply:
We can query for them in Graph API queries as far as I know. Or is the result incorrect?

Reply:
> in Graph API queries

I don't believe we can. We can query _by_ incoming links, but we can't actually get the incoming links

https://app.asana.com/0/1201095311341924/1203250435416416/f